cla_backend/apps/knowledgebase/tests/test_csv_2_fixture.py

```diff
@@ -12,9 +15,25 @@ class TestCSV2Fixture(TestCase):
     def setUp(self):
         self.datetime_now = datetime.now().replace(tzinfo=timezone.get_current_timezone()).isoformat()
 
-    def calculate_pk_range(self, article_category_matrices):
-        min_range = min(article_category_matrices, key=lambda x: x["pk"])
-        max_range = max(article_category_matrices, key=lambda x: x["pk"])
+    def find_path_to_csvfile(self, relative_path):
+        dirname = os.path.abspath(os.path.dirname(__file__))
+        csv_file = "CSVData/{}.csv".format(relative_path)
+        return os.path.join(dirname, csv_file)
+
+    def load_JSON_fixture_into_DB(self, relative_path):
+        try:
+            management.call_command("builddata", "load_knowledgebase_csv", self.find_path_to_csvfile(relative_path))
+            management.call_command(
+                "loaddata", os.path.abspath("cla_backend/apps/knowledgebase/fixtures/kb_from_spreadsheet.json")
+            )
+        except Exception as e:
+            self.fail(e)
+        else:
+            print("Record successfully loaded into DB!")
+
+    def calculate_pk_range(self, article_category_matrix):
```

Comment:
We discussed how this variable should probably either be plural or have something like `_list` in it